content/chainguard/chainctl-usage/getting-started-with-chainctl.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Getting Started with chainctl"
+lead: ""
+description: "chainctl basics for beginners"
+type: "article"
+date: 2025-03-0320T08:49:15+00:00
+lastmod: 2025-03-0320T08:49:15+00:00
+draft: false
+tags: ["chainctl", "Getting Started", "Product", "Basics"]
+images: []
+weight: 100
+---
+
+This page presents some of the more commonly used basic `chainctl` commands to help you get started. For a full reference of all commands with details and switches, see [chainctl Reference](../chainctl/_index.md).
+
+
+## Authenticate and Check Auth Status
```

Comment:
Maybe I'm thinking about this doc differently but I would include something like "Output Formats" that outlines different ways to show output from chainctl commands (`... -o [csv, id, json, none, table, terse, tree, wide]`)

Reply:
Good idea! Adding.